packages/contracts/test/integration/commitReveal.spec.ts

```diff
@@ -61,162 +62,176 @@ enum Player {
   GUESSING = 1
 }
 
-describe("CommitReveal", async () => {
-  const ConditionalTransfer = AbstractContract.loadBuildArtifact(
-    "ConditionalTransfer"
-  );
-  const StaticCall = AbstractContract.loadBuildArtifact("StaticCall");
-  const NonceRegistry = AbstractContract.loadBuildArtifact("NonceRegistry");
-  const Registry = AbstractContract.loadBuildArtifact("Registry");
-  const Signatures = AbstractContract.loadBuildArtifact("Signatures");
-  const Transfer = AbstractContract.loadBuildArtifact("Transfer");
-
-  const StateChannel = AbstractContract.loadBuildArtifact("StateChannel", {
-    StaticCall,
-    Signatures,
-    Transfer
+async function createMultisig() {
+  const multisig = new Multisig([alice.address, bob.address]);
+  await multisig.deploy(masterAccount);
+  await masterAccount.sendTransaction({
+    to: multisig.address,
+    value: parseEther("2")
```

Comment:
Maybe this helper function can be generalized to take an eth amount?

Reply:
Err I meant this comment for `createMultisig` so that the eth amount can be passed into `parseEther`

Reply:
Yea we could do that. The caller can pass in the variable, and we can use the same one to check `endBalanceA.sub(startBalanceA).should.be.bignumber.eq(parseEther("2"));` at the end of the test.

Reply:
In addition, it'd be changed the function signature to be:

`async function createMultisig(alice: PeerBalance, bob: PeerBalance)`, where

`interface PeerBalance { peer: Address; balance: number}`.

Then we could reuse this in other tests, e.g., in the machine.

Reply:
Yeah exactly

Reply:
Good suggestions. 

Side note: I believe every test should be idempotent, and therefore that each test should generate new Ethereum wallets with balance zero. This would make `startBalance*` obsolete. Likely I'll do this in a later PR.